walkthroughs/2-fuse-aggregator-and-api-management/walkthrough.adoc

```diff
@@ -557,11 +553,10 @@ Verify that you followed each step in the procedure above.  If you are still hav
 
 === Monitoring the API Service
 
-. Select the *Analytics* tab from the top menu.
-. Select *{fuse-flights-aggregator-app-name}* from the *Monitoring* section.
```

Comment:
It seems like "Monitoring" is not a section in the new list of analytics options in the UI, but the new options still allow a user to view requests